tests/sources/fixtures/azure_blob_storage/config.yml

```diff
@@ -26,6 +26,13 @@ service:
   job_cleanup_interval: 300
   log_level: INFO
 
+extraction_service:
+  enabled: false
```

Comment:
@seanstory Do you mean the other parts of the config here? e.g. `host` and `use_file_pointers` etc.

Reply:
I mean all of it. I would think that we could be able to leave out the whole `extraction_service` section, since `enabled: false` should be the default, and none of the other config should even be read in that case. I want to make sure that we won't force customers to configure and maintain these configs for a feature they explicitly don't want to use.

Reply:
That's a really good point and looks easy to fix. I'll make a PR for it.

Reply:
Follow-up PR: https://github.com/elastic/connectors-python/pull/1096